public/app/core/components/TransformersUI/rowsToFields/rowsToFields.ts

```diff
@@ -0,0 +1,193 @@
+import { map } from 'rxjs/operators';
+import {
+  anyToNumber,
+  ArrayVector,
+  DataFrame,
+  DataTransformerID,
+  DataTransformerInfo,
+  Field,
+  FieldColorModeId,
+  FieldConfig,
+  FieldType,
+  getFieldDisplayName,
+  ReducerID,
+  ThresholdsMode,
+} from '@grafana/data';
+
+export interface RowToFieldsTransformOptions {
+  nameField?: string;
+  valueField?: string;
+  mappings?: RowToFieldsTransformMappings[];
+}
+
+export interface RowToFieldsTransformMappings {
```

Comment:
Maybe:
```suggestion
export interface FieldToConfigMapping {
```

The row nature is not really true when a reducer is used.

Reply:
yea, been thinking about this rename as well :+1: